include/z64save.h

```diff
@@ -1595,6 +1595,9 @@ typedef enum {
     /* 3 */ DUNGEON_INDEX_STONE_TOWER_TEMPLE // Also applies to Inverted Stone Tower Temple
 } DungeonIndex;
 
+#define STRAY_FAIRY_TOTAL 25 // total number of stray fairies, including those already in the Great Fairy Fountain
+#define STRAY_FAIRY_SCATTERED_TOTAL 15 // original number of stray fairies in one dungeon area
```

Comment:
Could you use this macro here:
https://github.com/zeldaret/mm/blob/c6d3115fd3e9380476599fbe766a41a5a18b5b89/src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_map.c#L22